pkg/types/git_project.go

```diff
@@ -22,10 +29,30 @@ func (gp *GitProject) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return unmarshal((*raw)(gp))
 }
 
+// validateDirName evaluate directory name against forbidden characters
+func validateDirName(dirName string) bool {
+	return len(gitDirPatternNeg.Split(dirName, 2)) <= 1
```

Comment:
If you remove the `+` from the `gitDirPatternNeg`, you can simplify this check with `return gitDirPatternNeg.MatchString(dirName)`

Reply:
sure, that was first solution I had but later it started to fail with something.
will fix this..